File: backend/src/main/java/com/tripfriend/domain/trip/information/dto/TripInformationResDto.java
package com.tripfriend.domain.trip.information.dto;

import com.tripfriend.domain.trip.information.entity.Transportation;
import com.tripfriend.domain.trip.information.entity.TripInformation;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TripInformationResDto {
    private Long tripInformationId;
    private Long placeId;
    private String cityName;
    private String placeName;
    private LocalDateTime visitTime;
    private int duration;
    private Transportation transportation;
    private int cost;
    private String notes;
    //private int priority;
    private boolean isVisited;

    public TripInformationResDto(TripInformation tripInformation) {
        this.tripInformationId = tripInformation.getId();
        this.placeId = tripInformation.getPlace().getId();
        this.cityName = tripInformation.getPlace().getCityName();
        this.placeName = tripInformation.getPlace().getPlaceName();
        this.visitTime = tripInformation.getVisitTime();
        this.duration = tripInformation.getDuration();
        this.transportation = tripInformation.getTransportation();
        this.cost = tripInformation.getCost();
        this.notes = tripInformation.getNotes();
        //this.priority = tripInformation.getPriority();
        this.isVisited = tripInformation.isVisited();
    }
}

File: backend/src/main/java/com/tripfriend/domain/trip/information/dto/TripInformationResDto.kt
package com.tripfriend.domain.trip.information.dto

import com.tripfriend.domain.trip.information.entity.TripInformation


class TripInformationResDto(tripInformation: TripInformation) {
    var tripInformationId = tripInformation.id
    var placeId = tripInformation.place?.id
    var cityName = tripInformation.place?.cityName
    var placeName = tripInformation.place?.placeName
    var visitTime = tripInformation.visitTime
    var duration = tripInformation.duration
    var transportation = tripInformation.transportation
    var cost = tripInformation.cost
    var notes = tripInformation.notes
    var isVisited = tripInformation.isVisited
}
